src/app/content/components/Page.spec.tsx

```diff
@@ -566,6 +568,25 @@ describe('Page', () => {
     expect(dispatch).not.toHaveBeenCalled();
   });
 
+  it('noops if user decides not to discard', async() => {
```

Comment:
you should add this in the `src/app/content/components/Page/contentLinkHandler.spec.ts`, i'm trying not to add things to the page spec that could go in a more specific place because this spec file is so big